server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -33,6 +33,18 @@ import {
 } from "@fluidframework/server-services-telemetry";
 import { getBooleanFromConfig, getNumberFromConfig } from "./configUtils";
 
+interface IKeylessTokenClaims extends ITokenClaims {
+	/**
+	 * Identifies if the token is for Keyless Access or not.
+	 */
+	fluidRelayKeylessAccess?: boolean;
+}
+
+export function isKeylessFluidAccessClaimEnabled(token: string): boolean {
+	const claims = decode(token) as IKeylessTokenClaims;
```

Comment:
Here, you don't actually know if `token` is IKeylessTokenClaims or just ITokenClaims. See other comments where I propose alternative type and type guard usage